app-metrics/node_exporter/files/node_exporter-1.6.1.service

```diff
@@ -0,0 +1,21 @@
+[Unit]
+Description=Prometheus exporter for machine metrics
+Documentation=https://github.com/prometheus/node_exporter
+Requires=network-online.target
+After=network-online.target
+
+[Service]
+User=node_exporter
+Group=node_exporter
+Restart=on-failure
+Type=simple
```

Comment:
Doesn't seem too important for this service, but technically I think `Type=exec` is slightly more appropriate:
https://www.freedesktop.org/software/systemd/man/systemd.service.html#Type=

Reply:
Yeah `exec` makes more sense. Done